molo/commenting/admin.py

```diff
@@ -28,6 +45,16 @@ def reported_count(self, obj):
         return obj.flag_count(CommentFlag.SUGGEST_REMOVAL)
     reported_count.short_description = 'Times reported'
 
+    def comment_(self, obj):
+        # We only want to reply to root comments
+        if obj.parent is None:
+            reply_url = reverse(
+                'admin:commenting_molocomment_reply', args=(obj.id,))
+            return obj.comment + ' <a href="%s">(reply)</a>' % (reply_url)
```

Comment:
Bikeshed: I'm wondering whether the brackets should be outside of the `<a>` tag so that it renders like "([reply](#))" rather than "[(reply)](#)"? Perhaps it would be even better if it were a button? Does molo admin support bootstrap or similar?